consensus/ibft/state.go

```diff
@@ -97,12 +98,15 @@ func (c *currentState) setState(s IbftState) {
 }
 
 // NumValid returns the number of required messages
-func (c *currentState) NumValid() int {
-	// According to the IBFT spec, the number of valid messages
-	// needs to be 2F + 1
-	// The 1 missing from this equation is accounted for elsewhere
-	// (the current node tallying the messages will include its own message)
-	return 2 * c.validators.MaxFaultyNodes()
+func NumValid(validators ValidatorSet) int {
```

Comment:
What do you think about renaming this to `QuorumSize`?

Reply:
[Renamed](https://github.com/0xPolygon/polygon-edge/pull/513/commits/6e5f031b9e2f50cc8129606f41001c194b87c505)